core/src/main/scala/org/apache/spark/internal/config/Tests.scala

```diff
@@ -26,11 +26,11 @@ private[spark] object Tests {
     .longConf
     .createWithDefault(Runtime.getRuntime.maxMemory)
 
-  val TEST_SCHEDULE_INTERVAL =
-    ConfigBuilder("spark.testing.dynamicAllocation.scheduleInterval")
-      .version("2.3.0")
-      .longConf
-      .createWithDefault(100)
+  val TEST_DYNAMIC_ALLOCATION_SCHEDULE_ENABLED =
```

Comment:
when will we set this conf?

Reply:
When a new test needs it in the future...we don't need it now. Do you suggest to remove it?

Reply:
shall we turn it on by default then? I think we only need to disable it for one specific suite. This is also the behavior before this PR.

Reply:
ok